### PR TITLE
Master script addition 2

### PR DESCRIPTION
**Pull request by samaperrin**
# Why have changes been made?

Needed to flesh out the information about the page a little more. Have added and instructions, FAQ and contact page.

# What changes have been made?

- visualisation/hotspotMaps/data/textFunctions.R - Added functions for FAQs, contact, instructions and split up the landing page text
- visualisation/hotspotMaps/ui.R - added extra tabs to the include the txt written above

### CODE REVIEW

File: visualisation/hotspotMaps/data/textFunctions.R [hunk 1]
```diff
@@ -75,6 +87,235 @@ speciesIntensityText <- function() {
...
+      Luckily, GBIF will most likely have your species’ name floating around, and attached to the 
+      right species - but it will be listed as a synonym, and not the accepted scientific name. 
+      Make sure you use the accepted scientific name in focalSpecies.csv, or the pipeline will 
+      have trouble.")
+  )
+} 
+
+faqText8 <- function() {
+  mainPanel(
+    p("ANy decent spatial model needs to have some method of accounting for spatial autocorrelation -
```
Comment (PhilipMostert) [inline visualisation/hotspotMaps/data/textFunctions.R:269]: Lowercase n here

**Review by PhilipMostert — COMMENTED** [review]
Small error regarding capitalisation

**Comment by PhilipMostert** [comment]
Typing error in faqText8

File: visualisation/hotspotMaps/data/textFunctions.R (hunk 1 above)
Comment (samaperrin) [reply visualisation/hotspotMaps/data/textFunctions.R]: Fixed

**Review by PhilipMostert — APPROVED** [review]

**Pull request merged**